### ISSUE TITLE
npm install fail

### ISSUE THREAD

**Issue by khmyznikov** [nanostores/lit#8]
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @nanostores/lit@0.1.4
npm ERR! Found: nanostores@0.9.1
npm ERR! node_modules/nanostores
npm ERR!   nanostores@"^0.9.1" from the root project
npm ERR!   node_modules/@nanostores/router
npm ERR!     @nanostores/router@"^0.11.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer nanostores@"^0.7 || ^0.8" from @nanostores/lit@0.1.4
npm ERR! node_modules/@nanostores/lit
npm ERR!   @nanostores/lit@"^0.1.4" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: nanostores@0.8.1
npm ERR! node_modules/nanostores
npm ERR!   peer nanostores@"^0.7 || ^0.8" from @nanostores/lit@0.1.4
npm ERR!   node_modules/@nanostores/lit
npm ERR!     @nanostores/lit@"^0.1.4" from the root project
```

**Comment by akowi-sknobloch** [comment]
The Package needs to be updated to use the latest nanostores version.
There is already a pull request for that: #7 
I think @emilbonnek has to merge it

As a workaround you can use the `-f` option with your `npm install` to force the installation.
But be aware that force ignores all checks.

**Comment by emilbonnek** [comment]
Thank you 👍 
Closed by #7
